dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/query/QueryUtils.java

```diff
@@ -175,7 +175,8 @@ else if ( Collection.class.isAssignableFrom( klass ) )
             return (T) items;
         }
 
-        return null;
+        noCompatibleParserFound:
```

Comment:
Do we need this label here? I don't any place we use it.

Reply:
No, it's just to explicitly describe the reason we are throwing the exception at the end of the method. But I can remove if it doesn't fit the code standards. I will push this update in a few minutes.

Reply:
Thanks @stian-sandvold , change just pushed